.github/workflows/mintlify-broken-links.yml

```diff
@@ -0,0 +1,22 @@
+name: Mintlify Broken Links
+
+on:
+  push:
+    branches: [ '**' ]
```

Comment:
Don't think you need this. Removing it is the same as just running it on every push.

```suggestion
    branches: [ '**' ]
```

Reply:
haven't tried. Will do